Dockerfile

```diff
@@ -8,17 +8,22 @@ RUN mkdir -p /var/lib/ironic-inspector && \
     sqlite3 /var/lib/ironic-inspector/ironic-inspector.db "pragma journal_mode=wal"
 
 RUN crudini --set /etc/ironic-inspector/inspector.conf DEFAULT auth_strategy noauth && \ 
-    crudini --set /etc/ironic-inspector/inspector.conf ironic auth_strategy noauth && \
+    crudini --set /etc/ironic-inspector/inspector.conf ironic auth_type none && \
+    crudini --set /etc/ironic-inspector/inspector.conf service_catalog auth_type none && \
     crudini --set /etc/ironic-inspector/inspector.conf DEFAULT debug true && \
     crudini --set /etc/ironic-inspector/inspector.conf database connection sqlite:///var/lib/ironic-inspector/ironic-inspector.db && \
     crudini --set /etc/ironic-inspector/inspector.conf DEFAULT transport_url fake:// && \
+    crudini --set /etc/ironic-inspector/inspector.conf DEFAULT enable_mdns True && \
     crudini --set /etc/ironic-inspector/inspector.conf processing store_data database && \
     crudini --set /etc/ironic-inspector/inspector.conf processing ramdisk_logs_dir /shared/log/ironic-inspector/ramdisk && \
     crudini --set /etc/ironic-inspector/inspector.conf processing always_store_ramdisk_logs true && \
     crudini --set /etc/ironic-inspector/inspector.conf pxe_filter driver noop && \
     crudini --set /etc/ironic-inspector/inspector.conf processing node_not_found_hook enroll && \
     crudini --set /etc/ironic-inspector/inspector.conf discovery enroll_node_driver ipmi && \
-    crudini --set /etc/ironic-inspector/inspector.conf processing processing_hooks "\$default_processing_hooks,lldp_basic"
+    crudini --set /etc/ironic-inspector/inspector.conf processing processing_hooks "\$default_processing_hooks,lldp_basic" && \
+    # NOTE(dtantsur): keep this in sync with ironic-image/inspector.ipxe
+    # TODO(dtantsur): add ipa_inspection_collectors once we get ironic-lib 2.17.1
+    crudini --set /etc/ironic-inspector/inspector.conf mdns params 'ipa_debug:1,ipa_inspection_dhcp_all_interfaces:1,ipa_collect_lldp:1'
```

Comment:
inspector.ipxe uses ipa-inspection-dhcp-all-interfaces and ipa-collect-lldp (i.e. dashes not underscores), does it matter?

Reply:
Yep :( These are different because of the way they're processed on the IPA side.